utils/usbmuxd/files/usbmuxd.hotplug

```diff
@@ -0,0 +1,24 @@
+case "$ACTION" in
+	bind)
+		dev=/sys$DEVPATH
+
+		[ ! -f /tmp/iPhone.lock ] && [ -d ${dev}/net ] &&
+		{
+			readlink ${dev}/driver | grep ipheth &&
```

Comment:
shouldn't this use grep -q?

Reply:
This code runs without console and is not visible over the UART, so it shouldn't matter. It was left so I can see the output for debugging during development. I will change it.